### PR TITLE
Allowing soft tags to be excluded from tag count

### PR DESCRIPTION
**Pull request by tetondev**
Fixes [JIRA-4000](https://fhjira.churchofjesuschrist.org/browse/PS-4000)

- Simplifying alignment of text/add tag column
- Removing now unused CSS
- Subtracting softTag count from photoTagCount to provide a more accurate indication of tags to the user

### CODE REVIEW

File: fs-artifact-list-item.html [hunk 1]
```diff
@@ -915,9 +894,9 @@
         if (displayMessage) return true;
         return count === 0;
       },
-      _hideTag: function(count, displayMessage) {
+      _hideTag: function(data, displayMessage) {
         if (displayMessage) return true;
-        return count > 0;
+        return (data.photoTagCount - data.softTagCount) > 0;
```
Comment (rsnay) [inline fs-artifact-list-item.html:899]: I noticed the grid-item PR uses `data.softTagCount || 0`. Is that needed here as well?

Comment (tetondev) [reply fs-artifact-list-item.html:899]: It's probably not needed here...I just didn't know if the backend always gives us that prop. I will check with them. It wouldn't hurt to add it here, I guess

Comment (tetondev) [reply fs-artifact-list-item.html]: I added it to be safe

Comment (rsnay) [reply fs-artifact-list-item.html]: Makes sense 👍

**Pull request merged**